app/src/main/java/com/globant/example/mentorapp/home/presentation/view/fragment/ListUsersViewFragment.java

```diff
@@ -0,0 +1,154 @@
+import com.globant.example.mentorapp.home.presentation.presenter.ListUsersPresenterImpl;
+import com.globant.example.mentorapp.home.presentation.view.activity.HomeScreenActivity;
+import com.globant.example.mentorapp.home.presentation.view.adapter.ListUsersAdapter;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+/**
+ * A placeholder fragment containing a simple view.
+ */
+public class ListUsersViewFragment extends LifecycleFragment implements ListUsersViewInterface {
+
+    public SharedUserViewModel model;
+    @Inject
+    public ListUsersPresenterImpl presenter;
+    @Inject
+    protected LinearLayoutManager linearLayoutManager;
+    protected ListUsersAdapter listUsersAdapter;
+    @Inject
+    DividerItemDecoration itemDecoration;
+    private RecyclerView listUsersRecyclerView;
+    public static final String LIST_TAG = "listUserFragment";
+    HomeScreenActivity parentActivity;
+    public static boolean isFragmentAttached;
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ((MentorApplication) getActivity().getApplicationContext()).getApplicationComponent().inject(this);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_home_screen, container, false);
+        model = ViewModelProviders.of(this).get(SharedUserViewModel.class);
+        if (model.getUsers() != null || model.getUsers().getValue() != null) {
+            listUsersAdapter = new ListUsersAdapter(model.getUsers().getValue(), getContext());
+        }
+        listUsersRecyclerView = (RecyclerView) view.findViewById(R.id.list_item_container);
+        defineRecyclerView(listUsersRecyclerView, listUsersAdapter);
+        return view;
+    }
+
+    @Override
+    public void render(ListUsersViewModel usersModel) {
+        if (usersModel.getUsers() != null) {
+            model.setUsers(usersModel.getUsers());
+            usersReady();
+        } else switch (usersModel.getError()) {
+            case ERROR_CONNECTION:
+                usersErrorConnectivity();
+                break;
+            case ERROR_RESPONSE:
+                usersErrorHttp();
+                break;
+        }
+    }
+
+    public void usersReady() {
+        listUsersAdapter = new ListUsersAdapter(model.getUsers().getValue(), getContext());
+        listUsersRecyclerView.setAdapter(listUsersAdapter);
+        listUsersAdapter.notifyDataSetChanged();
+    }
+
+    private void usersErrorHttp() {
+        snackBarMessage(R.string.http_error_message);
+    }
+
+    private void usersErrorConnectivity() {
+        snackBarMessage(R.string.communication_error_message);
+    }
+
+    @Override
+    public void snackBarMessage(int stringResource) {
+        HomeScreenActivity.simpleSnackBarMessage(getString(stringResource));
```

Comment:
Make your activity use an interface, and invoke your snackbar through that interface